src/views/data/data-view-filters-container.js

```diff
@@ -107,6 +126,18 @@ export class DataViewFiltersContainer extends LitElement {
     this.updateFilters();
   }
 
+  handleFilesExcludeComboBoxChanged = e => {
+    let filenamesExclude = e.detail.value.map(item => item.filename);
+    this.selectedFilenamesExclude = filenamesExclude.map(item => '!' + item);
```

Comment:
please use template literals instead of string concatenation. https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Template_literals